src/main.cc

```diff
@@ -820,7 +820,20 @@ main(int argc, char **argv)
 			break;
 		case 'I':
 #if ENABLE_INTEGRITY
+#define CLEARENV_ERR "Environment is not reset, but integrity check is enabled."
```

Comment:
There is only one usage of this define. Is it better just to use this error message directly?

Reply:
OK, no problem. Fixed.
```diff
diff --git a/src/main.cc b/src/main.cc
index 2f15c54a78..42f3a68a2f 100644
--- a/src/main.cc
+++ b/src/main.cc
@@ -820,7 +820,6 @@ main(int argc, char **argv)
                        break;
                case 'I':
 #if ENABLE_INTEGRITY
-#define CLEARENV_ERR "Environment is not reset, but integrity check is enabled."
                        opt_mask |= O_INTEGRITY;
 
                        /*
@@ -830,7 +829,8 @@ main(int argc, char **argv)
                         * occurs, execution cannot be proceed.
                         */
                        if (clearenv() != 0) {
-                               fprintf(stderr, CLEARENV_ERR "\n");
+                               fprintf(stderr, "Environment is not reset, but "
+                                       "integrity check is enabled.\n");
                                return 1;
                        }
 
```